util/utils.js

```diff
@@ -1,15 +1,21 @@
+/* eslint-disable max-classes-per-file */
 class RespStatusError extends Error {
   constructor(message, statusCode) {
     super(message);
     this.statusCode = statusCode || 400;
   }
 }
 
+class RetryRequestError extends RespStatusError {
+  constructor(message) {
+    super(message, 809);
```

Comment:
809? what is the reason behind using this as status code?

Reply:
Server team needed an unique request code (not used by anyone). Randomly chosen

Reply:
Add comment so that it will be clear in the future